app_mesh/install.go

```diff
@@ -0,0 +1,80 @@
+package app_mesh
+
+import (
+	"fmt"
+
+	"github.com/layer5io/meshery-adapter-library/adapter"
+	"github.com/layer5io/meshery-adapter-library/status"
+
+	mesherykube "github.com/layer5io/meshkit/utils/kubernetes"
+)
+
+const (
+	repo  = "https://aws.github.io/eks-charts"
+	chart = "eks"
```

Comment:
```suggestion
	chart = "appmesh-controller"
```
`eks` is not name of the chart.

Reply:
yes thank you